lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -281,15 +281,16 @@ function paraneue_dosomething_preprocess_reportback_gallery(&$vars, $count = 6)
   drupal_add_js(
     [
       'dsKudosReactions' => [
+        'approved' => Kudos::getApprovedReactions(),
```

Comment:
Now we _only_ need to define the `approved` Kudos in the Kudos class property and that's it. One property defines the full approved listing!